jujugui/static/gui/src/app/components/user-profile/entity/test-entity.js

```diff
@@ -146,6 +146,7 @@ describe('UserProfileEntity', () => {
     const expected = (
```

Comment:
Yes we need a test for a case when UserProfileEntityKPI is rendered, and a case for an error while retrieving metrics with getKpiMetrics.